site/content/posts/2024-06-05-githubidentityprovider.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Pinniped v0.31.0: GitHub as an identity provider"
+slug: github-idp-support
+date: 2024-06-05
+author: Ryan Richard
+image: https://images.unsplash.com/photo-1557657043-23eec69b89c9?q=80&w=3008&auto=format&fit=crop&ixlib=rb-4.0.3
+excerpt: "With the release of v0.31.0, Pinniped brings GitHub identities to Kubernetes clusters everywhere"
+tags: ['Ryan Richard', 'release']
```

Comment:
Should we tag all the contributors?

Reply:
The tag only identifies the author of the blog post, not the authors/contributors of the code. It is not shown in the rendered blog post. It only decides what should be rendered on this page which you get to by clicking on my name: https://deploy-preview-1980--pinniped.netlify.app/tags/ryan-richard/